ivy/functional/backends/tensorflow/experimental/statistical.py

```diff
@@ -160,6 +161,125 @@ def corrcoef(
     return cor
 
 
+def set_value(ret):
+    if isinstance(ret, list):
+        return set_value(ret[0])
+    elif isinstance(ret, np.ndarray):
+        return set_value(ret[0])
+    else:
+        return ret
+
+
+def _nanmedian_helper(input, axis=None, keepdims=False):
+    dtype = input.dtype
+    temp = tf.cast(input, tf.float64)
+    num_dim = tf.rank(temp)
+    keepdim_shape = tf.shape(temp)
+    q = 50.0
+
+    if axis is None:
+        if tf.reduce_all(tf.math.is_nan(temp)).numpy():
+            temp = tf.reshape(temp, shape=(1, -1))
+        else:
+            temp = temp[~tf.math.is_nan(temp)]
+
+        ret = tfp.stats.percentile(
+            temp,
+            q,
+            axis=axis,
+            interpolation="midpoint",
+            keepdims=keepdims,
+        )
+        ret = tf.cast(ret, dtype=dtype)
+        return ret
+
+    if isinstance(axis, tuple):
+        if len(axis) == 1:
+            axis = axis[0]
+
```

Comment:
actually, I think we need to have this: 
```python
axis = [axis] if isinstance(axis, int) else list(axis)
```
since the previous check would cause an issue if an `int` is passed as an axis